packages/kbn-monaco/src/console/console_errors_provider.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { ConsoleWorkerProxyService } from './console_worker_proxy';
+import { CONSOLE_LANG_ID } from './constants';
+import { monaco } from '../monaco_imports';
+
+export const setupConsoleErrorsProvider = (workerProxyService: ConsoleWorkerProxyService) => {
```

Comment:
this file is copied from the previously used file `packages/kbn-monaco/src/ace_migration/setup_worker.ts`